avidemux_ru.ts

```diff
@@ -9883,7 +9883,7 @@ Ready.
     <message>
         <location line="-185"/>
         <source>Duplicate frames for a given duration.</source>
-        <translation>Повторять кадр в течение заданного времени.</translation>
+        <translation>Повторять кадр в течении заданного времени.</translation>
```

Comment:
A great update, thank you, just this one translation feels – no, is – wrong. We need a preposition here, not a prepositional case of a noun. I'm going to merge the PR and fix this afterwards. Thank you very much.